src/modules/i18n/locales/rm.json

```diff
@@ -81,7 +81,8 @@
 	"copy_success": "Copià il link!",
 	"copy_url": "Copiar il link",
 	"copyright_data": "© Datas:",
-	"copyright_label": "Copyright & decleraziun da protecziun da datas",
+	"copyright_label": "Nutzungsbedingungen",
```

Comment:
Line 437 I see that this has already been trasnslated in Rm as "cundiziuns d'utilisaziun", so you could take the same translation here.

Reply:
Good point

Reply:
Yes we could "vampirize" this translation, good point